packages/dd-trace/src/platform/node/index.js

```diff
@@ -55,3 +57,7 @@ const platform = {
 process.once('beforeExit', () => emitter.emit('exit'))
 
 module.exports = platform
+
+function hasSupportedAsyncLocalStorage () {
```

Comment:
Why is this a function instead of just a variable?

Reply:
Done. This is now just a variable.